plugins/woocommerce-beta-tester/src/live-branches/hooks/live-branches.tsx

```diff
@@ -39,3 +42,101 @@ export const useLiveBranchesData = () => {
 
 	return { branches, isLoading: loading };
 };
+
+export const useLiveBranchInstall = (
```

Comment:
This hook is basic and handles requests to manage the installation/activation of plugins